app/javascript/packs/webauthn-setup.ts

```diff
@@ -21,6 +21,11 @@ function webauthn() {
   }
   const continueButton = document.getElementById('continue-button')!;
   continueButton.addEventListener('click', () => {
+    // check if form is valid first
+    const form  = document.getElementById('webauthn_form') as HTMLFormElement;
+    if(!form.checkValidity()){
+      return;
+    }
```

Comment:
One thing I noticed in my work in #7803 ([related comment](https://github.com/18F/identity-idp/pull/7803/files#r1103237487)) is that the form `submit` event seems to be a reliable way to handle things _after_ validation. Maybe we could shuffle this around so that the "Continue" button is treated as a proper submit button?

```suggestion
  const form  = document.getElementById('webauthn_form') as HTMLFormElement;
  form.addEventListener('submit', () => {
```